### DISCUSSION TITLE
automating-ecs-deployment-by-github-actions/

### DISCUSSION THREAD

**Discussion by giscus** [Hyeon9mak/Hyeon9mak.github.io#167] [Blog comments]
# automating-ecs-deployment-by-github-actions/

이미 ECS상에 배포환경(클러스터, 서비스, 태스크 정의)이 모두 준비된 상태에서,  애플리케이션의 변화가 일어났을 때 Github actions를 이용해  자동으로 배포를 진행하는(태스크 내부 애플리케이션 컨테이너를 교체하는) 자동화 방법을 경험해보았다.  

https://hyeon9mak.github.io/automating-ecs-deployment-by-github-actions/

**Comment by junu0810** [comment]
좋은 내용확인하고 도움받고 갑니다. 감사합니다.
